tools/FhirSchemaManager/Commands/ApplyCommand.cs

```diff
@@ -38,7 +37,7 @@ public static async Task HandlerAsync(string connectionString, Uri fhirServer, i
                     availableVersions.RemoveAt(0);
                 }
 
-                availableVersions = availableVersions.Where(availableVersion => availableVersion.Id <= version)
+                availableVersions = availableVersions.Where(availableVersion => availableVersion.Id <= (next == true ? availableVersions.First().Id : version))
```

Comment:
nit: simplify into a variable `maxVersion` maybe?

Reply:
In the latest version PR, the version is moved to MutualExclusiveType class and then accessed as MutualExclusiveType.version.